runbotics/runbotics-orchestrator-ui/src/src-app/translations/pl/process/process.json

```diff
@@ -341,6 +341,7 @@
     "Process.Details.Modeler.Actions.CloudExcel.GetRange.Label": "Pobierz zakres",
     "Process.Details.Modeler.Actions.CloudExcel.GetRange.WorksheetName": "Nazwa arkusza",
     "Process.Details.Modeler.Actions.CloudExcel.GetRange.WorksheetName.Info": "To pole jest opcjonalne. Jeśli nie zostanie określone, nazwa arkusza będzie taka sama, jak podana w akcji 'Otwórz skoroszyt' lub domyślna",
+    "Process.Details.Modeler.Actions.CloudExcel.GetRange.VariableName.Info": "Przypisuje wynik do zmiennej. Zwracana wartość to obiekt zawierający pola startCell, endCell, ColumCount, rowCount, cellCount oraz tekst = zawartość arkusza zwrócona jako tablica tablic, np. [['A1', 'B1', 'C1'], ['A2', 'B2', 'C2']]",
```

Comment:
it's a details but CloumnCount is in PascalCase instead of camelCase

Reply:
and there's no "tekst" but "text" property right? 

also i suggest to replace "=" with "(zawartość arkusza zwrócona jako tablica tablic, np. [['A1', 'B1', 'C1'], ['A2', 'B2', 'C2']])"